### PR TITLE
pbTests: Test script fixes

### PR DESCRIPTION
**Pull request by Willsparker**
Additional fixes to :
1) stop `testJDK.sh` finding the `debug-image` thats produced when building JDKs

2) Allow `testJDKWin.sh` to find built JDKs other than JDK8

### CODE REVIEW

**Comment by Willsparker** [comment]
Latest commit ensures JDK8 is built with JDK8 on Windows. When running the build with JDK_BOOT_DIR set to jdk7, it attempts to run Gradle with it, causing Gradle to fail. 
https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/452/OS=Win2012,label=infra-vagrant-1/console

File: ansible/pbTestScripts/buildJDKWin.sh [hunk 1]
```diff
@@ -63,7 +63,7 @@ checkJDKVersion() {
         case "$jdk" in
                 "jdk8u" | "jdk8" | "8" | "8u" )
                         JAVA_TO_BUILD="jdk8u";
-			JDK_BOOT_DIR=/cygdrive/c/openjdk/jdk7;;
```
Comment (sxa555) [inline ansible/pbTestScripts/buildJDKWin.sh]: Why is this being changed from 7 to 8?

**Comment by karianna** [comment]
What version of gradle dropped 7 support and do we need to execute gradle as part of our boot strapping?

**Comment by sxa555** [comment]
Normally JAVA_HOME is set to 8 (or it's in the PATH by default) and we just set the JDK boot dir to the boot one, which should be 7. We don't want to change JDK8 to bootstrap with 8 - it's not what we do in the production builds, and so it would be wrong to do that here.

**Comment by Willsparker** [comment]
Okay, I'll put it back and just ensure `JAVA_HOME` is jdk8

**Comment by sxa555** [comment]
Can you do a VPC run to ensure this is ok @Willsparker ?

**Comment by Willsparker** [comment]
https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/456/
I'll do a jdk8 run now too

**Pull request merged**